skypy/galaxy/_spectrum_loaders.py

```diff
@@ -16,6 +16,13 @@
 except ImportError:
     pass
 
+# this file is only ever imported when speclite is present
+# but without the try/except pytest will fail when doctests are discovered
+try:
+    import speclite.filters
+except ImportError:
+    pass
+
 
 def download_file(url, cache=True):
```

Comment:
Is this function still needed?

Reply:
Technically yes the kcorrect data is still downloaded from skypy-data (which is being addressed separately in #379 )

Reply:
not downloaded, used from `skypy-data` unless I'm misremembering things?

Reply:
Ah yes that's true, sorry, it is only the decam filters that were downloaded then? In which case this can go.